src/UNL/UCBCN/Manager/Controller.php

```diff
@@ -0,0 +1,170 @@
+<?php
+/**
+ * This is the primary viewing interface for the events.
+ * This would be the 'model/controller' if you follow that paradigm.
+ *
+ * This file contains functions used throughout the frontend views.
+ *
+ * PHP version 5
+ *
+ * @category  Events
+ * @package   UNL_UCBCN_Manger
+ * @author    Tyler Lemburg <lemburg@unl.edu>
+ * @copyright 2015 Regents of the University of Nebraska
+ * @license   http://www1.unl.edu/wdn/wiki/Software_License BSD License
+ * @link      http://code.google.com/p/unl-event-publisher/
+ */
+namespace UNL\UCBCN\Manager;
+
+use UNL\UCBCN\RuntimeException;
+use UNL\UCBCN\UnexpectedValueException;
+
+class Controller {
+    public $options = array(
+        'format' => 'html',
+        'model' => false
+    );
+
+    /**
+     * Configurable ID for the base/master calendar
+     *
+     * @var int
+     */
+    public static $default_calendar_id = 1;
+
+    public function __construct($options = array()) {
+        $this->options = $options + $this->options;
+
+        try {
+            $this->run();
+        } catch (\Exception $e) {
+            $this->output = $e;
+        }
+
+    }
+
+    public static $url = '/manager/';
+
+    public static $web_root = '/';
+
+    /**
+     * Get the URL to the manager
+     *
+     * @return string
+     */
+    public function getURL()
+    {
+        return self::$url;
+    }
+
+    public function getWebRoot() {
```

Comment:
I can't because RegExpRouter needs that for a baseURL. We can refactor later if need be, but this is fine for starting and it's sort of 'base' code that just needs to work.